gcp_variant_transforms/beam_io/vcfio.py

```diff
@@ -883,6 +884,66 @@ def expand(self, pvalue):
     return pvalue.pipeline | Read(self._source)
 
 
+def _create_vcf_source(
+    file_pattern=None, compression_type=None, allow_malformed_records=None):
+  return _VcfSource(file_pattern=file_pattern,
+                    compression_type=compression_type,
+                    allow_malformed_records=allow_malformed_records)
+
+
+class ReadAllFromVcf(PTransform):
+  """A :class:`~apache_beam.transforms.ptransform.PTransform` for reading a
+  :class:`~apache_beam.pvalue.PCollection` of VCF files.
+
+  Reads a :class:`~apache_beam.pvalue.PCollection` of VCF files or file patterns
+  and produces a PCollection :class:`Variant` (or
+  :class:`MalformedVcfRecord for failed reads) objects.
+
+  This transform should be used when reading from massive (>70,000) number of
+  files.
+  """
+
+  DEFAULT_DESIRED_BUNDLE_SIZE = 64 * 1024 * 1024  # 64MB
+
+  def __init__(
+      self,
+      min_bundle_size=0,
```

Comment:
Currently min_bundle_size is not configurable in _VcfSource. I think we should remove it as a parameter here and always set it to 0 since that is what it will be set to by default in FileBasedSource.

Reply:
Good point. Done.